modules/contact/test/tests/pdass_problems/hertz_cyl_half_1deg.jou

```diff
@@ -88,7 +88,7 @@ mesh surf all
 
```

Comment:
Does this journal file correspond to the mesh used in the test?

Reply:
You're right I reverted the mesh but not the journal file, good catch!